ecosystem/python/sdk/aptos_sdk/async_client.py

```diff
@@ -780,19 +780,22 @@ class FaucetClient:
     base_url: str
     rest_client: RestClient
 
-    def __init__(self, base_url: str, rest_client: RestClient):
+    def __init__(self, base_url: str, rest_client: RestClient, auth_token: str = None):
```

Comment:
hopefully this actually fails the python liniting, but this should be `Optional[str]`